src/containers/Game/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { useContext, useEffect, useState } from 'react';
+
+import { DIFFICULTY_LEVEL_VALUE, ROUTES, SUCCESS_INCREASE_DIFFICULTY_FACTOR } from './../../constants';
+import DictionaryContext from './../../context/DictionaryContext';
+import { convertSecondsToMMSS, generateWord, getHighestScoreObj, getItemFromStorage } from './../../utils/helpers';
+
+import Button from './../../components/Button';
+import CurrentGameDetails from './../../components/CurrentGameDetails';
+import PlayingArea from './../../components/PlayingArea';
+import ScoreBoard from './../../components/ScoreBoard';
+import Icon from './../../elements/Icon';
+
+import closeIcon from './../../assets/images/icons/close.svg';
+import gamepadIcon from './../../assets/images/icons/gamepad.svg';
+import homeIcon from './../../assets/images/icons/home.svg';
+import userIcon from './../../assets/images/icons/user.svg';
+
+import './style.scss';
+
+const Game = ({ history }) => {
+  const dictionary = useContext(DictionaryContext);
+  const [name, setName] = useState("");
+  const [difficultyLevel, setDifficultyLevel] = useState("");
+  const [difficultyFactor, setDifficultyFactor] = useState(1);
+  const [word, setWord] = useState("");
+  const [isGameOver, setIsGameOver] = useState(false);
+  const [score, setScore] = useState(null);
+  const [scoreBoard, setScoreBoard] = useState([]);
+  const [currentGameObj, setCurrentGameObj] = useState(null);
+  const [isNewHighScore, setIsNewHighScore] = useState(false);
+
+  useEffect(() => {
+    const name = getItemFromStorage("name");
+    const difficultyLevel = getItemFromStorage("difficultyLevel");
+    setName(name);
+    setDifficultyLevel(difficultyLevel);
+    setDifficultyFactor(DIFFICULTY_LEVEL_VALUE[difficultyLevel]);
```

Comment:
Use function instead of `HashMap/object` and inside function use `switch - case`

Reply:
Can you please describe what exactly is wrong/not-good here?